src/Juvix/Compiler/Concrete/Pretty/Base.hs

```diff
@@ -222,6 +222,11 @@ instance PrettyCode OperatorArity where
       AssocLeft -> kwInfixl
       AssocNone -> kwInfix
 
+instance PrettyCode SyntaxDef where
+  ppCode s =
+    (kwSyntax <+>) <$> case s of
```

Comment:
since we store the `_opSyntaxKw` in `SyntaxOperator` we should print it in that instance and not here

Reply:
done